src/benchmarks/gc/README.md

```diff
@@ -179,6 +140,21 @@ Each trace file can be opened in PerfView if you need to.
 Each trace file will be named `{coreclr_name}__{config_name}__{benchmark_name}__{iteration}`, e.g.  `clr_a__smaller__nosurvive__0`.
 
 
+## Test status files
+
+Each trace has (at least) two files associated with it, `x.etl` (or `x.nettrace`) and `x.yaml`.
+The `.yaml` file is called a test status file. It provides, among other things, the process ID to focus on.
+A minimal test status file looks like:
+
+    # this file: `x.yaml`
+    success: true
+    trace_file_name: x.etl  # Should generally match the name of this file
```

Comment:
does trace_file_name take a full path of x.etl or just x.etl? if it's the latter does it mean the .yaml always needs to be the in same dir as x.etl?

Reply:
Added clarifying comment.